app/assets/stylesheets/application.scss

```diff
@@ -152,6 +194,19 @@
   background-color: #5bc0de !important;
```

Comment:
Not gonna lie, I saw `!important` and nearly gasped… but it looks like you're using it on utility classes, which is [a great use case for `!important`](https://css-tricks.com/when-using-important-is-the-right-choice/)! Bravo! However, I don't think this particularly case (`.timeline-panel:before`) counts as a utility class so I'd find a different solution here.